functions-and-operators/string-functions.md

```diff
@@ -147,11 +147,105 @@ Return the index position of the first argument within the second argument.
 
 ### [`FORMAT()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_format)
 
-Return a number formatted to specified number of decimal places.
+The `FORMAT()` function is used to round numbers and format them to a specified number of decimal places, returning a number formatted to the specified decimal places.
+
+The function rounds and formats based on the specified number of decimal places. If the specified number of decimal places is greater than the actual number of decimal places in the number, the result is padded with zeros to the corresponding length.
+
+**Parameters:**
+
+1. **Number (or Numeric String):** The number to be formatted. It can be a direct numeric value, a numeric string, or a number in scientific notation.
+2. **Decimal Places:** The specified number of decimal places for the returned value.
```

Comment:
```suggestion
- **Number (or Numeric String):** The number to be formatted. It can be a direct numeric value, a numeric string, or a number in scientific notation.
- **Decimal Places:** The specified number of decimal places for the returned value.
```